arviz/plots/tsplot.py

```diff
@@ -0,0 +1,430 @@
+"""Plot timeseries data."""
+import warnings
+import numpy as np
+
+from ..sel_utils import xarray_var_iter
+from ..rcparams import rcParams
+from .plot_utils import default_grid, get_plotting_function
+
+
+def plot_ts(
+    idata,
+    y,
+    x=None,
+    y_hat=None,
+    y_holdout=None,
+    y_forecasts=None,
+    x_holdout=None,
+    plot_dim=None,
+    holdout_dim=None,
+    num_samples=100,
+    backend=None,
+    backend_kwargs=None,
+    y_kwargs=None,
+    y_hat_plot_kwargs=None,
+    y_mean_plot_kwargs=None,
+    vline_kwargs=None,
+    textsize=None,
+    figsize=None,
+    legend=True,
+    grid=False,
+    axes=None,
+    show=None,
+):
+    """Plot timeseries and it's components.
+
+    Parameters
+    ----------
+    idata : InferenceData
+    y : str, variable name from observed_data
+    x : str, Optional
+        If none, coords of y dims
+    y_hat : str, from posterior_predictive, optional
+        Assumed to be of shape (chain, draw, *y.dims)
+    y_holdout : str, from observed_data, optional
+        Observed data after holdout
+    y_forecasts : str, from posterior_predictive, optional
+        Assumed shape (chain, draw, *)
+    x_holdout : str, from constant_data or y_holdout coords, optional
+        If None, coords of y_holdout or coords of y_forecast is chosen.
+    plot_dim: str, Optional
+        Necessary to choose x if x is None & if y is multidimensional.
+    holdout_dim: str, Optional
+        Necessary to choose x_holdout if x is None
+        and if y_holdout or y_forecasts is multidimensional.
+    num_samples : int, Optional, Default 100
```

Comment:
```suggestion
    num_samples : int, default 100
```

in numpydoc style, using the `default xxx` idiom already indicates it's an optional argument because if it's not passed it will fall back to the default. The convention is also to use lowercase

Reply:
Oh, I miss that! I was over-concerned about the functionality and forgot to double-check the docstring! Thanks for pointing it out!